tamr_toolbox/utils/operation.py

```diff
@@ -128,3 +128,31 @@ def wait(
             return operation
         operation = operation.poll()
     raise TimeoutError(f"Waiting for operation took longer than {timeout_seconds} seconds.")
+
+
+def monitor(
+    operation: Operation, *, poll_interval_seconds: int = 1, timeout_seconds: Optional[int] = 300,
```

Comment:
With latest default, change type for `timeout_seconds`  to `int` instead of `Optional[int]`

Reply:
Great catch. Done